pkgs/os-specific/linux/zfs/unstable.nix

```diff
@@ -16,17 +16,16 @@ callPackage ./generic.nix args {
     else kernel.kernelOlder "6.2";
 
   latestCompatibleLinuxPackages = if stdenv'.isx86_64 || removeLinuxDRM
-    then linuxKernel.packages.linux_6_4
-    else linuxKernel.packages.linux_6_1;
+  then linuxKernel.packages.linux_6_4
+  else linuxKernel.packages.linux_6_1;
```

Comment:
This is inconsistent with the rest of the file

Reply:
Why are you nitpicking a formatting inconsistency 3 weeks after a PR was merged?

Reply:
Because I only stumbled upon this PR yesterday when I dug up my old zfsUnstable bump PR.